i18nilize/build/lib/internationalize/helpers.py

```diff
@@ -0,0 +1,132 @@
+import json
+import sys
+
```

Comment:
remove this old logic, we wont be using this anymore

Reply:
still needed